generators/reference_chunking/newline_splitter/__init__.py

```diff
@@ -0,0 +1,22 @@
+from pydantic import BaseModel
+
+INPUT_EXAMPLE = {
+    "text": """This is the first line.
+    And this is the second line.
+    Here's a third one, too.
```

Comment:
this text holds spaces at the beginning of the line. Is this wanted? 
![image](https://github.com/code-kern-ai/bricks/assets/91723236/242523d3-6fb0-49ef-a857-55e6d6cd6fc0)

Reply:
No, but this can be avoided by using `.strip()` on the returned values instead of the original text. This should remove all leading and tailing whitespaces.